base/Dockerfile

```diff
@@ -7,4 +7,4 @@ RUN curl -s https://coduno.github.io/cli/install.sh | bash -s - -y
 
 WORKDIR /run
 
-ENTRYPOINT ["/usr/local/bin/coduno", "run"]
+ENTRYPOINT ["/bin/bash", "-c", "coduno prepare > /dev/null && coduno run"]
```

Comment:
Redirect is okay, but to some file that we can store, in case `coduno prepare` fails (for example malformed commands, wrong `apt-get` invocation ...).